docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -126,116 +136,363 @@ observers of sex, based on gender assigned at birth (14); multiple
+ACS PUMS rows in the specified PUMAs with replacement, and with
+sampling weights given by ACS data; here is sample code for a nanosim
+initial population:
+
+.. sourcecode:: python
+
+    # load some ACS data
+    columns = ['household_id', 'location', 'fips code', 'puma', 
+               'weight', 'age', 'sex', 'race_eth', 'relshipp',
+               'mig', 'migpuma', 'migsp']
+    acs = pd.read_csv('/home/j/Project/Models/VEHSS/prepped/acs_2019_pums.csv', low_memory=False, usecols=columns)
+    acs_hh_only = acs[acs.household_id.str.contains('HU')]  # subset of rows for "household" sample, meaning those _not_ in group quarters
+
+    # sample households to initialize population table
+    n_households = 3
+
+    p = acs_hh_only.query(location_str).groupby('household_id').weight.mean() # FIXME: load and use household weights here, instead of this
+    p /= p.sum()
+
+    resampled_households = np.random.choice(a=p.index, p=p,
+                                            size=n_households, replace=True)
+
+    g = acs.groupby('household_id')
+    def household(i, hh_id):
+        dfg = g.get_group(hh_id).copy()
+        dfg['household_id'] = i
+        return dfg
+    df_population = pd.concat([household(i, hh_id) for i, hh_id in enumerate(resampled_households)])
+
+In the code above, there is a location string filter which we can use
+to focus our simulation on a single state or PUMA.  For our initial
+model, please focus on Florida, with
+
+.. sourcecode:: python
+
+    location_str = 'location == "FL"'  # restrict to subset of ACS data, e.g. specific state or PUMA
+
+Here is a small example of what the code in this section will load from ACS:
+
++---------+---------------+-------+------+-----------+------+-----------+-----------+-------------+
+|         | household_id  | puma  | age  | relshipp  | sex  | race_eth  | location  | fips code   |
++=========+===============+=======+======+===========+======+===========+===========+=============+
+| 801679  | 0             | 1110  | 5    | 25        | 1    | 2         | FL        | 12          |
++---------+---------------+-------+------+-----------+------+-----------+-----------+-------------+
+| 801678  | 0             | 1110  | 39   | 20        | 2    | 2         | FL        | 12          |
++---------+---------------+-------+------+-----------+------+-----------+-----------+-------------+
+| 782698  | 1             | 7301  | 67   | 20        | 2    | 1         | FL        | 12          |
++---------+---------------+-------+------+-----------+------+-----------+-----------+-------------+
+| 782699  | 1             | 7301  | 82   | 36        | 1    | 1         | FL        | 12          |
++---------+---------------+-------+------+-----------+------+-----------+-----------+-------------+
+| 801484  | 2             | 12703 | 82   | 20        | 1    | 1         | FL        | 12          |
++---------+---------------+-------+------+-----------+------+-----------+-----------+-------------+
```

Comment:
Is there a reason you're using hardcoded tabular tables instead of easier-to-edit `list-table`s?